### PR TITLE
Migration all modules to 8.0

### PR DESCRIPTION
**Pull request by mileo**

### CODE REVIEW

**Comment by max3903** [comment]
CLA problem fixed.

**Comment by bwrsandman** [comment]
Rebase is needed

File: account_fiscal_position_rule_purchase/purchase.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        ctx.update({'use_domain': ('use_purchase', '=', True)})
+        return self.env['account.fiscal.position.rule'].with_context(
+            ctx).apply_fiscal_mapping(result, **kwargs)
+
+    @api.multi
+    def onchange_partner_id(self, partner_id):
+        ctx = dict(self._context)
+        result = super(PurchaseOrder, self).onchange_partner_id(partner_id)
+
+        print "onchange_partner_id"
```
Comment (renatonlima) [inline account_fiscal_position_rule_purchase/purchase.py]: print function in code, I fixed this in original migration branch https://github.com/akretion/account-fiscal-rule/tree/feature/account_fiscal_position_rule_purchase_migration_8.0 and made one PR to integrate this branch in 8.0 branch project #11

**Comment by rvalyi** [comment]
Hello just like for the localization, we will migrate modules 1 by 1 ensuring a proper test coverage all the way. There are already separate PR's for all modules and they also contain a generalization of the web-context-tunnel module so that the overrides of native on-change method don't have their signature changed ensuring a better compatibility with other modules and customzations. So propose to close this PR and rather construct a unified work branch from the separate PR's rather than proposing these unified work branches to be merged.

**Comment by rvalyi** [comment]
so :-1: for me

**Pull request closed without merging**